src/main/java/uk/gov/companieshouse/search/api/service/search/impl/alphabetical/AlphabeticalSearchRequestService.java

```diff
@@ -101,6 +101,8 @@
                 topHitCompany.setCompanyName(company.getCompanyName());
                 topHitCompany.setCompanyNumber(company.getCompanyNumber());
                 topHitCompany.setCompanyStatus(company.getCompanyStatus());
+                topHitCompany.setCompanyType(company.getCompanyType());
+                topHitCompany.setLinks(company.getLinks());
                 topHitCompany.setOrderedAlphaKeyWithId(company.getOrderedAlphaKeyWithId());
                 topHitCompany.setKind(ALPHABETICAL_KIND);
```

Comment:
We may want to consider moving this to the Mapper class as it's mapping the top hit for Alphabetical searches. Not necessarily as part of this PR though, just something to think about.